backend/src/packages/orders/order.entity.ts

```diff
@@ -95,12 +120,14 @@ class OrderEntity implements IEntity {
     status,
     userId,
     businessId,
-    driverId,
+    shiftId,
     customerName,
     customerPhone,
-  }: Omit<OrderEntityT, 'id'>): OrderEntity {
+  }: Omit<OrderEntityT, 'shift' | 'driver' | 'truck'> & {
+    shiftId: number;
+  }): OrderEntity {
     return new OrderEntity({
-      id: null,
+      id,
```

Comment:
To Pavlo's comment on necessity of passing id.
```suggestion
      id: null
```

Reply:
I've commented this case above